source/text/FontSet.h

```diff
@@ -13,18 +13,20 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #ifndef ES_TEXT_FONT_SET_H_
 #define ES_TEXT_FONT_SET_H_
 
-#include <string>
+#include "Font.h"
```

Comment:
Was the change from forward declaration to include required? Consuming classes will definitely need to have the Font include if they use its methods, but if they are just passing the reference from ::Get to some other class they would be able to do without it.